src/components/content-types/TextContent.tsx

```diff
@@ -328,23 +371,38 @@ function TextInList(props: EditableContentProps) {
         title={title}
         documentId={documentId}
         editable={editable}
+        onBlur={onBlur}
+        color={
+          unseenChanges ? "var(--colorBlack)" : "var(--colorSecondaryGrey)"
+        }
       />
     </ListItem>
   )
 }
 
-export const hasUnseenChanges = memoize(
-  (doc: Doc<unknown>, lastSeenHeads?: LastSeenHeads) => {
-    return getUnseenPatches(doc, lastSeenHeads).some(
-      (patch) =>
-        patch.action === "splice" &&
-        patch.path.length === 2 &&
-        patch.path[0] === "text"
-    )
-  },
-  (doc, lastSeenHeads) =>
-    `${getHeads(doc).join(",")}:${JSON.stringify(lastSeenHeads)}`
-)
+function hasUnseenChanges(doc: Doc<unknown>, lastSeenHeads?: LastSeenHeads) {
+  return getUnseenPatches(doc, lastSeenHeads).some(
+    (patch) =>
+      patch.action === "splice" &&
+      patch.path.length === 2 &&
+      patch.path[0] === "text"
+  )
+}
+
+// TODO: this is not really checking if the user has been mentioned since the doc was last seen
```

Comment:
Just to confirm, is the plan to defer this todo for now?

Reply:
I think we should wait until we get support for marks. Then, we can replace mentions with a custom object. That will make checking if you have unseen mentions trivial.